Remove unsupported utf-8

milljm opened the pull request:
Use decode to convert bytes to string (and string to string for
python2).

Closes #14248

Commit:
Remove unsupported utf-8

Use decode to convert bytes to string (and string to string for
python2).

Closes #14248

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/418198/) on 5fb7a91 wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/14249/coverage)

Commit:
Verify command used encoding

Remove utf-8 encoding from subprocess command.

Refs #14249

Commit:
Additional subprocess's with encoding='utf-8'

Remove the encoding attribute from subprocess methods.

Refs #14249

Commit:
Load yaml, not pyyaml

This test is always being skipped due to incorrect module name.

Not exactly related to Python-3 PR, but I only discovered it during
extensive tests against this branch.

Refs #14249

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/418424/) on e43aa98 wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/14249/coverage)

Commit:
Fix warnings as errors for GCC 9

Refs #14249

moosebuild commented:
Job [Code Coverage](https://civet.inl.gov/job/418528/) on f5bd64b wanted to post the following:

View code coverage site [here](https://mooseframework.inl.gov/docs/PRs/14249/coverage)
